File: Applications.GenuisReader/GeniusReader.Domain/EFModel/Series.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;

namespace GeniusReader.Domain.EFModel
{
    [Table("Series", Schema = "dbo")]
    public class Series
    {
        public int SeriesId { get; set; }
        public string Name { get; set; }
        public bool IsOngoing { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime? EndDate { get; set; }
        public List<Author> Authors { get; set; }
        public List<Tag> Tags { get; set; }
        public List<Chapter> Chapters { get; set; }
    }
}

File: Applications.GenuisReader/Applications.GenuisReader/ClientApp/src/app/models/series.ts
import { Author, blankAuthor } from "./author";
import { Chapter } from "./chapter";
import { Tag } from "./tag";

export class Series {
    public seriesId: number;
    public title: String;
    public startDate: Date;
    public endDate: Date|null;
    public authors: Author[];
    public tags: Tag[];
    public isOngoing: Boolean;
    
    constructor(seriesId: number, title: String, startDate: Date, endDate: Date, authors: Author[], tags: Tag[], ongoing: Boolean, public chapters: Chapter[]) {
        this.seriesId = seriesId;
        this.title = title;
        this.startDate = startDate;
        this.endDate = endDate;
        this.authors = authors;
        this.tags = tags;
        this.isOngoing = ongoing;
    }
}

export const blankSeries: Series  = {
    seriesId: 0,
    title: 'Blank',
    startDate: new Date(1900, 0, 1),
    endDate: null,
    authors: [blankAuthor],
    tags: [],
    isOngoing: false,
    chapters: [],
}